### PR TITLE
fix error docs

### PR DESCRIPTION
**Pull request by buuug7**
line 68 , the `.remove()` should changed to `.detach()`

### CODE REVIEW

**Comment by buuug7** [comment]
mybe nobody to maintain this repository, closed this PR
